ci/travis_script_python.sh

```diff
@@ -164,6 +168,10 @@ export PYARROW_BUILD_TYPE=$ARROW_BUILD_TYPE
 export PYARROW_WITH_PARQUET=1
 export PYARROW_WITH_PLASMA=1
 export PYARROW_WITH_ORC=1
+export PYARROW_WITH_S3=1
+if [ "$ARROW_TRAVIS_S3" == "1" ]; then
+  export PYARROW_WITH_S3=1
```

Comment:
Hmm... I suppose there's a leftover here or above?

Reply:
ARROW_TRAVIS_S3 is still optional, so we need it.

Reply:
Yes, but look at the diff :-)

Reply:
What do you mean, have I removed ARROW_TRAVIS_S3 or PYARROW_WITH_S3?

Reply:
I mean this:
```
export PYARROW_WITH_S3=1
if [ "$ARROW_TRAVIS_S3" == "1" ]; then
  export PYARROW_WITH_S3=1
```

Reply:
:), fixing it